core/domain/param_domain.py

```diff
@@ -86,18 +86,17 @@ class ParamChange(object):
 
     def __init__(self, name, generator_id, customization_args):
         """Initialize a ParamChange object with the specified arguments.
-
         Args:
             name: unicode. The name of the parameter.
-                generator_id: unicode. The type of generator used to create the
-                parameter, e.g., "Copier".
+            generator_id: unicode. The type of generator used to create the
+            parameter, e.g., "Copier".
```

Comment:
Please indent the continuation lines by 4. Only 'name', 'generator_id' and 'customization_args' should be fully left-aligned.